src/site/layouts/step_by_step.drupal.liquid

```diff
@@ -60,22 +60,7 @@
                     {{ fieldStep.entity.fieldWysiwyg.processed }}
                     <br />
 
-                    <!-- Step alert -->
-                    {% if fieldStep.entity.fieldAlert.length %}
-                      {% for alert in fieldStep.entity.fieldAlert %}
-                        {% if alert.entity != empty %}
-                          <div class="vads-u-margin-bottom--4">
-                            {% include "src/site/blocks/alert.drupal.liquid" with alert = alert.entity %}
-                          </div>
-                        {% endif %}
-                      {% endfor %}
-                    {% else %}
-                      {% if fieldStep.entity.fieldAlert.entity != empty %}
-                        <div class="vads-u-margin-bottom--4">
-                          {% include "src/site/blocks/alert.drupal.liquid" with alert = fieldStep.entity.fieldAlert.entity %}
-                        </div>
-                      {% endif %}
-                    {% endif %}
+                    {% include "src/site/paragraphs/alert_single.drupal.liquid" with alert = fieldStep.entity.fieldAlert.entity %}
```

Comment:
Hmm, what if there are multiple alerts? Looks like the previous code was trying to support that but not sure if it matters

Reply:
I'm not sure - so many data shapes. But the `alert_single` matches the `entityBundle` for the `fieldStep.entity.fieldAlert.entity` so I think this is correct